frontend/src/components/ShootHibernation/HibernationScheduleEvent.vue

```diff
@@ -77,16 +77,17 @@ SPDX-License-Identifier: Apache-2.0
 </template>
 
 <script>
-import { getValidationErrors } from '@/utils'
-import { required, requiredIf } from 'vuelidate/lib/validators'
-import moment from 'moment-timezone'
 import join from 'lodash/join'
 import split from 'lodash/split'
 import get from 'lodash/get'
 import map from 'lodash/map'
 import find from 'lodash/find'
 import isEqual from 'lodash/isEqual'
 import sortBy from 'lodash/sortBy'
+import { required, requiredIf } from 'vuelidate/lib/validators'
+
+import { getValidationErrors } from '@/utils'
+import moment from '@/utils/moment'
```

Comment:
When editing an existing hibernation schedule, it always shows the current time when the dialog was opened instead of the configured `wake up` and `hibernate at` values 
<img width="707" alt="Screenshot 2021-04-13 at 23 07 06" src="https://user-images.githubusercontent.com/5526658/114621629-378a0700-9cad-11eb-9b78-027b9d9274b1.png">

Reply:
and another bug on the shoot details page. The creation date is showing the value `lll`. Only the tooltip is correct.

<img width="459" alt="Screenshot 2021-04-13 at 23 14 16" src="https://user-images.githubusercontent.com/5526658/114622467-46bd8480-9cae-11eb-8e4e-4745ef4d4897.png">

Reply:
Added `localizedFormat` plugin to support `lll` format.

Reply:
dayjs does not change the date object if you call `hour` or `minute` but returns the new modified object instead. We have to chain all setters because of this.

```js
// old code
const momentObj = moment().utc()
momentObj.hour(13)
momentObj.minute(27)
console.log(momentObj.format())
// new code
const dayjsObj = moment().utc()
  .hour(13)
  .minute(27)
console.log(dayjsObj.format())
```